v1/lib/static/css/main.css

```diff
@@ -1680,7 +1680,7 @@ input#search_input_react {
     position: -webkit-sticky;
     position: sticky;
     overflow-y: auto;
-    top: 50px; /* Height of navbar */
+    top: 50px; / Height of navbar /
```

Comment:
Ehh... this is definitely not correct css syntax.

I guess we can just remove the comment entirely? I'm surprised docusaurus doesn't like the comment, since similar ones are used elsewhere: https://github.com/aragon/Docusaurus/blob/da8d7bd2d45642093895e6eec9d2290bf5440e0b/v1/lib/static/css/main.css#L1566.